src/amo/components/OverallRating.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+
+import translate from 'core/i18n/translate';
+
+import 'amo/css/OverallRating.scss';
+
+
+export class OverallRatingBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object,
+    addonName: PropTypes.string,
+  }
+
+  render() {
+    const { i18n, addonName } = this.props;
+    const prompt = i18n.sprintf(
+      i18n.gettext('How are you enjoying your experience with %(addonName)s?'),
+      { addonName });
+
+    return (
+      <div className="OverallRating">
+        <p>{prompt}</p>
+        <form action="">
+          <input id="love-it" value="love-it" name="overall-rating" type="radio" />
+          <label htmlFor="love-it" className="love-it">
+            {i18n.gettext('Love it!')}
+          </label>
+          <input id="it-is-ok" value="it-is-ok" name="overall-rating" type="radio" />
```

Comment:
Nothing wrong with making the relationship between label and input explicit like this. I've always personally preferred using attributes instead of relying on the implied relationship even when wrapping the input with a label.